src/util.h

```diff
@@ -779,10 +780,40 @@ inline v8::MaybeLocal<v8::Value> ToV8Value(v8::Local<v8::Context> context,
         .Check();                                                              \
   } while (0)
 
+// Variation on NODE_DEFINE_CONSTANT that sets a Symbol value.
+#define NODE_DEFINE_SYMBOL_CONSTANT(target, name, constant)                    \
+  do {                                                                         \
+    v8::Isolate* isolate = target->GetIsolate();                               \
+    v8::Local<v8::String> constant_name =                                      \
+        v8::String::NewFromUtf8(isolate, name).ToLocalChecked();               \
+    v8::Local<v8::Symbol> constant_value =                                     \
+        v8::Symbol::ForApi(isolate,                                            \
+            v8::String::NewFromUtf8(isolate, constant).ToLocalChecked());      \
+    v8::PropertyAttribute constant_attributes =                                \
+        static_cast<v8::PropertyAttribute>(v8::ReadOnly | v8::DontDelete);     \
+    target                                                                     \
+        ->DefineOwnProperty(isolate->GetCurrentContext(),                      \
+                            constant_name,                                     \
+                            constant_value,                                    \
+                            constant_attributes)                               \
+        .Check();                                                              \
+  } while (0)
+
 constexpr inline bool IsLittleEndian() {
   return std::endian::native == std::endian::little;
 }
 
+enum class Endianness { LITTLE, BIG };
+
+inline Endianness GetEndianness() {
```

Comment:
We already have a method for endianness, but if you need to know you should use

https://en.cppreference.com/w/cpp/types/endian

Reply:
that one got here from a merge from main branch, going to check what happened to it, shouldnt show up in a diff